faq.html

```diff
@@ -155,7 +155,7 @@ <h4 class="panel-title">
               </div>
               <div id="F-1" class="panel-collapse collapse">
                 <div class="panel-body">
-                  Apparently, you are using a build not coming from the Google Play Store. These builds don't contain our Flattr API key.
+                  Apparently, you did not install AntennaPod via the Google Play Store. Unfortunately, we can't provide our Flattr API key (and thus Flattr integration) in other builds.
```

Comment:
"You didn't install AntennaPod from the Google Play Store. Your build doesn't contain the necessary credentials for Flattr integration to work."?